spaceshooter/__init.py__

```diff
@@ -0,0 +1,6 @@
+## initialize pygame and create window
+pygame.init()
+pygame.mixer.init()  ## For sound
+screen = pygame.display.set_mode((WIDTH, HEIGHT))
+pygame.display.set_caption("Space Shooter")
+clock = pygame.time.Clock()     ## For syncing the FPS
```

Comment:
If someone runs `python <path to>/spaceShooter.py` what result will be? It looks like this code is not required because it does nothing in this case.

Plus: where do you have constants `WIDTH` and `HEIGHT` from? Where is `pygame` import?

Looks like total nonsense and should be removed.